slug.go

```diff
@@ -34,14 +34,71 @@ func (e *IllegalSlugError) Error() string {
 // chain.
 func (e *IllegalSlugError) Unwrap() error { return e.Err }
 
+// PackerOption is a functional option that can configure non-default Packers.
+type PackerOption func(*Packer) error
+
+// ApplyIgnore is a PackerOption that will apply the .terraformignore rules and
+// skip packing files it specifies.
+func ApplyIgnore() PackerOption {
```

Comment:
Nitpick: I'd kind of like this better as `ApplyTerraformIgnore`. It's more explicit and obvious what it does.